runtime/testruntime/testruntime.go

```diff
@@ -224,11 +228,21 @@ func NewInstanceForProject(t TestingT, name string) (*runtime.Runtime, string) {
 	return rt, inst.ID
 }
 
-func NewInstanceForDruidProject(t TestingT) (*runtime.Runtime, string) {
+func NewInstanceForDruidProject(t *testing.T) (*runtime.Runtime, string, error) {
+	if os.Getenv("RILL_RUNTIME_DRUID_TEST_DSN") == "" {
+		t.Skip("skipping the test without the test instance")
+	}
+
 	rt := New(t)
 
 	_, currentFile, _, _ := goruntime.Caller(0)
 	projectPath := filepath.Join(currentFile, "..", "testdata", "ad_bids_druid")
+	var err error
+	b64 := os.Getenv("RILL_RUNTIME_DRUID_TEST_DSN")
+	dsn, err := base64.StdEncoding.DecodeString(b64)
+	if err != nil {
+		return nil, "", err
+	}
```

Comment:
Why does it need to be base64 encoded? We have other env variables containing DSNs that are not.

Reply:
It contains password in format https://username:password@hostname...
Bash has special symbols that the password can contain, to reduce the mental burden remembering which ones, I just coded it into base64.

Reply:
I thought bash/env variables can contain any ASCII string? Just need to escape it or use single quotes to enter it. 

Given that all other environment variables in the project do not use base64, I think we should avoid it here as well.

Reply:
Let's try it.